components/http_foundation/introduction.rst

```diff
@@ -250,8 +250,8 @@ by using the following methods:
 :method:`Symfony\\Component\\HttpFoundation\\Request::getCharsets`
     Returns the list of accepted charsets ordered by descending quality.
 
-* :method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`:
-  returns the list of accepted charsets ordered by descending quality;
+:method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`
+    Returns the list of accepted charsets ordered by descending quality.
```

Comment:
We have to fix the sentence too (seems be copied from the one above). We are not talking about charsets here.